tests/test-web-purchase.js

```diff
@@ -285,7 +312,7 @@ describe('fxpay.pay.acceptPayMessage()', function() {
     );
   });
 
-  it('had window closed by user', function(done) {
+  it('had window closed by user on unload', function(done) {
```

Comment:
nit: this would read better to me as: ``it('handles window closed by user via unload...``